ImageFeed/ImageList/ImageListViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  ImageListViewController.swift
+//  ImageFeed
+//
+//  Created by Егор Свистушкин on 23.01.2023.
+//
+
+import UIKit
+
+class ImageListViewController: UIViewController {
+
+
+    // MARK: - Outlets
+    @IBOutlet private var tableView: UITableView!
+    
+    
+    
+    
+    // MARK: - Properties (var & let)
```

Comment:
обычно всем понятно что это вар и лет :)
```suggestion
    // MARK: - Properties 
```

Reply:
Я для себя)